### ISSUE TITLE
List public information about projects with public API

### ISSUE THREAD

**Issue by madprime** [OpenHumans/open-humans#719]
The public API should have an endpoint that lists public information about projects.

In explanation, this is information about projects that is publicly available on activity pages, for projects that have been "approved" (i.e. not all projects in development).

One use-case for this endpoint is mapping project identifiers (e.g. "direct-sharing-5") to a project name (e.g. "Genevieve Genome Report").

The URL for this could be `/api/public-data/projects/`

**Comment by gedankenstuecke** [comment]
Actually, is there right now any way to get this mapping from project identifiers to their name? (Writing the notebooks it always felt like cheating when having to look over the user objects)

**Comment by madprime** [comment]
If you're using notebooks or any other project with access tokens (master or OAuth2) there's this endpoint: https://www.openhumans.org/direct-sharing/on-site-data-access/#api-project-info but it's only for the project itself. It's frustrating to get something like "direct-sharing-123" and not know how to map it, right?

**Comment by gedankenstuecke** [comment]
Yeah, e.g. for the notebooks that doesn't help in finding out which of the 100 files returned in the `user['data']` list is the Twitter archive, Fitbit data etc. So I always look over each element in the `data` list until I hit the right `source` 😂
